specification/network/resource-manager/Microsoft.Network/stable/2020-03-01/privateEndpoint.json

```diff
@@ -396,6 +396,206 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Network/privateEndpoints/{privateEndpointName}/privateDnsZoneGroups/{privateDnsZoneGroupName}": {
```

Comment:
>/privateDnsZoneGroups/{privateDnsZoneGroupName} [](start = 138, length = 47)

you are missing a collection GET that lists all privateDnsZoneGroups under the privateEndpoint (GET ..../privateEndpoints/{privateEndpointName}/privateDnsZoneGroups)

Reply:
will this just be a list GET by specifying both subscription and resourcegroup, or just subscription?

Reply:
or both?

Reply:
It will specify subscription, resource group, and the parent privateEndpointName.  You will only be listing all the privateDnsZoneGroups in a privateEndpoint

Reply:
Pending, will push soon after confirming child resource issue.